sdk/openai/Azure.AI.OpenAI/src/Custom/AzureOpenAIClientOptions.cs

```diff
@@ -1,17 +1,43 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+using System.ClientModel.Primitives;
 using OpenAI;
 
 namespace Azure.AI.OpenAI;
 
 /// <summary>
 /// Defines the scenario-independent, client-level options for the Azure-specific OpenAI client.
 /// </summary>
-public partial class AzureOpenAIClientOptions : OpenAIClientOptions
+public partial class AzureOpenAIClientOptions : ClientPipelineOptions
 {
-    internal string Version => _version;
-    private readonly string _version;
+    internal string Version { get; }
+
+    /// <summary>
+    /// Specifies custom authorization scopes to use when authenticating a client with a supplied <see cref="Azure.Core.TokenCredential"/>.
+    /// </summary>
+    public IReadOnlyList<string> TokenAuthorizationScopes
```

Comment:
Is this how these are typically specified in Azure clients?  @christothes, @schaabs can you confirm?

Reply:
It's unusual - typically the scope is constant and well known for the resource. 

@trrwilson - What are the possible values in this case and when would a customer need to specify them?

Reply:
Updated to an extensible enum, currently with just public/government (per PM discussion on explicit support) but also with the capability to use a string directly for the authorization scope.